tsconfig.json

```diff
@@ -17,23 +17,43 @@
       "dom"
     ],
     "strictPropertyInitialization": false,
-    "strict": true,
+    "strict": false,
```

Comment:
I will revert it back when I work on the `strick` ticket.

Reply:
It's probably safer to just disable it at the lib level.